docs/conf.py

```diff
@@ -21,7 +21,7 @@
 author = 'Gustav Pettersson @ ESA'
 
 # The full version, including alpha/beta/rc tags
-release = '0.1'
+release = '0.1.0'
```

Comment:
releases will be via git tags in the `*.*.*` format so its best to always use the full version string.